src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -384,4 +384,18 @@ public void setCreated_NonProduction(Date createdAt) {
     public void setUpdatedAt_NonProduction(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
+
+    /**
+     * Checks whether the edit form of student has changed the section/team values.
+     * @param originalStudentAttribute
+     * @return true if either section or team has changed from previous values. 
+     */
+    public boolean hasSectionOrTeamChanged(StudentAttributes originalStudentAttribute) {
+       if ((this.section!=null && !this.section.equals(originalStudentAttribute.section)) || 
```

Comment:
Do ensure that there are spaces surrounding boolean operators, i.e. `this.section != null`. Address this in other areas of the changed code as well :)